### PR TITLE
996 task - test for the Sidedrawer component

### PR DESCRIPTION
**Pull request by himanshudube97**
- This has the test for the sidedrawer component.

### CODE REVIEW

File: src/components/SideDrawer/__tests__/SideDrawer.test.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      pathname: '/',
+      push,
+    });
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  expect(sideMenu).toBeDefined();
+  expect(sideMenu.length).toBeGreaterThan(0);
```
Comment (mdshamoon) [inline src/components/SideDrawer/__tests__/SideDrawer.test.tsx:34]: Add all expectations inside a test case.

File: src/components/SideDrawer/__tests__/SideDrawer.test.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+  beforeEach(() => {
```
Comment (mdshamoon) [inline src/components/SideDrawer/__tests__/SideDrawer.test.tsx:36]: Use a single beforeEach function for a test file

**Review by mdshamoon — CHANGES_REQUESTED** [review]

File: src/components/SideDrawer/__tests__/SideDrawer.test.tsx (hunk 2 above)
Comment (himanshudube97) [reply src/components/SideDrawer/__tests__/SideDrawer.test.tsx:36]: ok. I ll check that. Thanks.

Comment (himanshudube97) [reply src/components/SideDrawer/__tests__/SideDrawer.test.tsx]: Added single beforeEach done.

File: src/components/SideDrawer/__tests__/SideDrawer.test.tsx (hunk 1 above)
Comment (himanshudube97) [reply src/components/SideDrawer/__tests__/SideDrawer.test.tsx]: - created a seperated test case both this.

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**